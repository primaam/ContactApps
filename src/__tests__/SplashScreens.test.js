import React from 'react';
import {ActivityIndicator, Text} from 'react-native';
import renderer from 'react-test-renderer';

// it('renders correctly', () => {
//   const tree = renderer.create(<SplashScreens />).toJSON();
//   expect(tree).toMatchSnapshot();
// });

it('renders the ActivityIndicator component', () => {
  const tree = renderer.create(<ActivityIndicator size="large" />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders the Text component', () => {
  const tree = renderer.create(<Text>Please Wait</Text>).toJSON();
  expect(tree).toMatchSnapshot();
});
