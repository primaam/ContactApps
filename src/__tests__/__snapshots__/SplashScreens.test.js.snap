// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ActivityIndicator component 1`] = `
<ActivityIndicator
  size="large"
/>
`;

exports[`renders the Text component 1`] = `
<Text>
  Please Wait
</Text>
`;
